htmresearch/frameworks/pytorch/cnn_sdr.py

```diff
@@ -124,6 +129,8 @@ def __init__(self,
     shape = self.outputSize()
```

Comment:
Don't you have to change outputSize() too?

Reply:
`outputSize` was already using stride parameter